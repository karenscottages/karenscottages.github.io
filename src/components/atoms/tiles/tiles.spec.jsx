import React from 'react';
import renderer from 'react-test-renderer';
import { Tile } from './';

describe("Atoms/Tiles", () => {
    describe("Tile", () => {
        it('still renders correctly', () => {
            const tree = renderer.
                create(<Tile>Example Tile</Tile>).
                toJSON();
            expect(tree).toMatchSnapshot();
        });
    });

    describe("Tile.Ancestor", () => {
        it('still renders correctly', () => {
            const tree = renderer.
                create(<Tile.Ancestor>Example Tile</Tile.Ancestor>).
                toJSON();
            expect(tree).toMatchSnapshot();
        });
    });

    describe("Tile.Parent", () => {
        it('still renders correctly', () => {
            const tree = renderer.
                create(<Tile.Parent>Example Tile</Tile.Parent>).
                toJSON();
            expect(tree).toMatchSnapshot();
        });
    });

    describe("Tile.Child", () => {
        it('still renders correctly', () => {
            const tree = renderer.
                create(<Tile.Child>Example Tile</Tile.Child>).
                toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});