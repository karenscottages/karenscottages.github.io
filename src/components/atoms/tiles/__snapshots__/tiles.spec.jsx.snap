// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Tiles Tile still renders correctly 1`] = `
<tile
  className="tile "
>
  Example Tile
</tile>
`;

exports[`Atoms/Tiles Tile.Ancestor still renders correctly 1`] = `
<tile
  className="tile is-ancestor undefined"
>
  Example Tile
</tile>
`;

exports[`Atoms/Tiles Tile.Child still renders correctly 1`] = `
<tile
  className="tile is-child undefined"
>
  Example Tile
</tile>
`;

exports[`Atoms/Tiles Tile.Parent still renders correctly 1`] = `
<tile
  className="tile is-parent undefined"
>
  Example Tile
</tile>
`;
